src/app/authorization/auth-component/auth.component.ts

```diff
@@ -0,0 +1,40 @@
+import {Component, OnInit} from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import {FormControl, FormGroup, Validators} from '@angular/forms';
```

Comment:
adjust IDE to auto add spaces
```suggestion
import { Component, OnInit } from '@angular/core';
import { HttpClient } from '@angular/common/http';
import { FormControl, FormGroup, Validators } from '@angular/forms';
```